build.gradle

```diff
@@ -203,14 +216,20 @@ task generateInstallers(type: Install4jTask, dependsOn: [renameShadowJar, copyJR
     }
 }
 
-task setRunnableLinux(dependsOn: [generateInstallers]) {
+task prepareInstallers(group: 'release', dependsOn: [generateInstallers]) {
     doLast {
-        def linuxFile = "TripleA_${version}_unix.sh"
-        ant.chmod(dir: "$buildDir/releases", perm: '+x', includes: linuxFile)
+        def releasesDir = "$buildDir/releases"
+
+        ant.chmod(dir: releasesDir, perm: '+x', includes: "TripleA_${version}_unix.sh")
```

Comment:
That's pretty neat!
I knew the installer name scheme could be changed (I did this myself in a PR a couple days ago) bit I didn't know this name scheme could be changed for the individual installers